netci.groovy

```diff
@@ -198,14 +204,14 @@ static void addVsiArchive(def myJob) {
 
 // ISSUE: Temporary until a full builder for multi-scm source control is available.
 // Replace the scm settings with a multiScm setup.  Note: for PR jobs; explicitly set the refspec
-static void addVsiMultiScm(def myJob, def project) {
+static void addVsiMultiScm(def myJob, def project, def isPullR) {
```

Comment:
Definitely don't do this. `isPullR` here is a new variable, different from the global `isPullR`. This won't compile.